NOTICE

```diff
@@ -0,0 +1,7 @@
+Copyright 2024 Finschia Foundation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Sorry are these not necessary?

- https://github.com/Finschia/cosmwasm/blob/main/NOTICE
- https://github.com/Finschia/finschia/blob/main/NOTICE

```
   Unless required by applicable law or agreed to in writing, software
   distributed under the License is distributed on an "AS IS" BASIS,
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   See the License for the specific language governing permissions and
   limitations under the License.
```

Reply:
Thank you. I added it.